client/layout/guided-tours/positioning.js

```diff
@@ -32,25 +32,25 @@ const helpers = {
 	yBelow: ( bottom ) => {
 		return bottom + DIALOG_PADDING;
 	},
-	xAboveBelow: ( left, right ) => {
+	xAboveBelow: ( left, right, width ) => {
 		if ( ( left + DIALOG_WIDTH + DIALOG_PADDING ) < document.documentElement.clientWidth ) {
 			return left + DIALOG_PADDING;
 		} else if ( right - DIALOG_WIDTH - DIALOG_PADDING > 0 ) {
-			return right - DIALOG_WIDTH - DIALOG_PADDING;
+			return right - ( DIALOG_WIDTH - width );
 		}
 		return DIALOG_PADDING;
 	},
 };
 
 const dialogPositioners = {
 	below: ( rect ) => {
-		const x = helpers.xAboveBelow( rect.left, rect.right );
+		const x = helpers.xAboveBelow( rect.left, rect.right, rect.width );
```

Comment:
Simplest solution might be to disable this tour for non-desktops and create an issue for the framework. I think we should really implement measuring of the step and make this positioning _great again_.

![width](https://cloud.githubusercontent.com/assets/156676/20933054/22b0f570-bbd7-11e6-9c2a-9c41b556ec33.gif)